events/firehose.go

```diff
@@ -32,3 +34,10 @@ type KinesisFirehoseResponseRecord struct {
 	Result   string `json:"result"` // The status of the transformation. May be TransformedStateOk, TransformedStateDropped or TransformedStateProcessingFailed
 	Data     []byte `json:"data"`
 }
+
+type KinesisFirehoseRecordMetadata struct {
+	ShardID                     string                `json:"shardId"`
+	PartitionKey                string                `json:"partitionKey"`
+	SequenceNumber              string                `json:"sequenceNumber"`
+	ApproximateArrivalTimestamp MilliSecondsEpochTime `json:"approximateArrivalTimestamp"`
```

Comment:
The CI failed (https://travis-ci.org/aws/aws-lambda-go/builds/530382884). Need to check if the type needs to be `string`, or if the test data is wrong.
```
...
--- FAIL: TestSampleTransformation (0.00s)
    firehose_test.go:50: could not unmarshal event. details: json: cannot unmarshal string into Go struct field KinesisFirehoseRecordMetadata.approximateArrivalTimestamp of type int64
...
```

Reply:
found a reference internally and a sample event in the console - I will update the test data and merge